src/CalculatorUnitTests/DateCalculatorUnitTests.cpp

```diff
@@ -387,6 +387,65 @@ namespace DateCalculationUnitTests
             VERIFY_IS_TRUE(StringReference(L"") != viewModel->StrDateResult);
         }
 
+        TEST_METHOD(DateCalcViewModelDateDiffDaylightSavingTimeTest)
+        {
+          auto viewModel = ref new DateCalculatorViewModel();
+
+          viewModel->IsDateDiffMode = true;
+          VERIFY_IS_TRUE(viewModel->IsDateDiffMode);
+
+          /*29.02.2008*/
```

Comment:
I'm not sure what date format calc normally uses internally, but i'd encourage YYYY-MM-DD as opposed to DD-MM-YYYY. The standard US format is MM-DD-YYYY and thus using DD-MM-YYYY is fairly unexpected and at times ambiguous. Since no one writes YYYY-DD-MM, using YYYY-MM-DD is unambiguous and fairly easy for people to understand.